css/style.css

```diff
@@ -902,6 +910,12 @@ button.button-inline:hover {
 	}
 }
 
+@media (max-width: 768px) {
+	#app-navigation-toggle {
+		display: none !important;
```

Comment:
One more thing that needs to be fixed is that the button should also be visible on the boards list overview ;)